src/Tribe/Attendee_Registration/Template.php

```diff
@@ -104,6 +104,17 @@ public function is_on_ar_page() {
 		return tribe( 'tickets.attendee_registration' )->is_on_page();
 	}
 
+	/**
+	 * Convenience wrapper for tribe( 'tickets.attendee_registration' )->is_using_shortcode() usage.
+	 *
+	 * @since TBD
+	 *
+	 * @return boolean
```

Comment:
```suggestion
	 * @return boolean Whether the Attendee Registration shortcode is being used.
```

Reply:
thanks! fixed that in https://github.com/the-events-calendar/event-tickets/pull/1919